Running make in conformance-checkers spits up in environments where python2 is default

In my system, I've set the default python version to 2 for compatibility with some tools I need daily.

When I run `make`, I get this error:
```python
python  -mjson.tool --sort-keys messages.json > messages.json.tmp
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib64/python2.7/json/tool.py", line 40, in <module>
    main()
  File "/usr/lib64/python2.7/json/tool.py", line 24, in main
    infile = open(sys.argv[1], 'rb')
IOError: [Errno 2] No such file or directory: '--sort-keys'
make: *** [Makefile:33: messages.json] Error 1
```

I fixed it via:

```diff
diff --git a/conformance-checkers/Makefile b/conformance-checkers/Makefile
index f7774341cb..a7ed7bcfae 100644
--- a/conformance-checkers/Makefile
+++ b/conformance-checkers/Makefile
@@ -7,7 +7,7 @@ EXPAND=expand
 EXPANDFLAGS=
 GIT=git
 GITFLAGS=
-PYTHON=python
+PYTHON=python3
 PYTHONFLAGS=
 CURL=curl
 CURLFLAGS=
```

Not sure if that is the fix you want, or if you'd prefer error handling.

As an aside, 

```sh
$ PYTHON=python3 make
```

doesn't work because `Makefile` stomps on the correct version and restores the system default version.

--- a/conformance-checkers/Makefile
+++ b/conformance-checkers/Makefile
@@ -1,20 +1,20 @@
-HTML2MARKDOWN=html2text
-PERL=perl
-PERLFLAGS=
-FMT=fmt
-FMTFLAGS=-80
-EXPAND=expand
-EXPANDFLAGS=
-GIT=git
-GITFLAGS=
-PYTHON=python
-PYTHONFLAGS=
-CURL=curl
-CURLFLAGS=
-JAVA=java
-JAVAFLAGS=
-VNU_TEST_REPO=git@github.com:validator/tests.git
-ITS_REPO=git@github.com:w3c/its-2.0-testsuite-inputdata.git
+HTML2MARKDOWN ?= html2text
+PERL ?= perl
+PERLFLAGS ?=
+FMT ?= fmt
+FMTFLAGS ?= -80
+EXPAND ?= expand
+EXPANDFLAGS ?=
+GIT? = git
+GITFLAGS ?=
+PYTHON ?= python3
+PYTHONFLAGS ?=
+CURL ?= curl
+CURLFLAGS ?=
+JAVA ?= java
+JAVAFLAGS ?=
+VNU_TEST_REPO ?= git@github.com:validator/tests.git
+ITS_REPO ?= git@github.com:w3c/its-2.0-testsuite-inputdata.git
 .PHONY: .FORCE
 
 all: README.md messages.json

